packages/client/components/PokerEstimateHeaderCardJira.tsx

```diff
@@ -31,7 +31,7 @@ const HeaderCard = styled('div')({
 const CardTitle = styled('h1')({
   fontSize: 16,
   lineHeight: '24px',
-  margin: 0
+  marginBottom: '10px'
```

Comment:
Material Design defaults to an 8px grid in most cases. This is probably a good place to use 1x the grid at 8px.